docs/content/references/cli/move.mdx

```diff
@@ -16,13 +16,13 @@ Usage: sui move [OPTIONS] <COMMAND>
 
 Commands:
   build
-  coverage 	Inspect test coverage for this package. A previous test run with the `--coverage` flag must have previously been run
+  coverage 	  Inspect test coverage for this package. A previous test run with the `--coverage` flag must have previously been run
   disassemble
-  new      	Create a new Move package with name `name` at `path`. If `path` is not provided the package will be created in the directory `name`
-  prove    	Run the Move Prover on the package at `path`. If no path is provided defaults to current directory. Use `.. prove .. -- <options>` to pass on options to the
-               	prover
-  test     	Run Move unit tests in this package
-  help     	Print this message or the help of the given subcommand(s)
+  test     	  Run Move unit tests in this package
```

Comment:
test should go between `new` and `help`. 
<img width="398" alt="image" src="https://github.com/user-attachments/assets/602de104-0df9-4d53-b41d-a7a90f474caa">